idaes_examples/mod/diagnostics/gas_solid_contactors/properties/methane_iron_OC_reduction/gas_phase_thermo.py

```diff
@@ -0,0 +1,780 @@
+##############################################################################
```

Comment:
Do the gas-solid contactor files need to be duplicated here, or could they just be imported from https://github.com/IDAES/idaes-pse/tree/main/idaes/models_extra/gas_solid_contactors?

Reply:
The files need to be duplicated. They are ported from IDAES 1.7, and include a "broken" version of the model, so they should not be distributed with the current version of IDAES. I also don't think it's a good idea (a) enforce that this notebook be run with IDAES 1.7 or (b) somehow import these file directly from 1.7.